leakcanary-android/src/main/java/com/squareup/leakcanary/AndroidExcludedRefs.java

```diff
@@ -452,7 +453,7 @@
   },
 
   SYSTEM_SENSOR_MANAGER__MAPPCONTEXTIMPL((LENOVO.equals(MANUFACTURER) && SDK_INT == KITKAT) //
-      || (VIVO.equals(MANUFACTURER) && SDK_INT == LOLLIPOP_MR1)) {
+      || (VIVO.equals(MANUFACTURER) && SDK_INT == LOLLIPOP_MR1) || (WIKO.equals(MANUFACTURER) && SDK_INT == KITKAT)) {
```

Comment:
Might be better to group all manufacturers together. Also, I wonder if this is really manufacturer specific or if there was an actual leak in AOSP? Weird to see it in different manufacturers